tomaty/tomaty_serialization.py

```diff
@@ -0,0 +1,60 @@
+"""
+tomaty_serialization.py
+~~~~~~~~~~~~~
+
+basic serialization to preserve effort on any given day.
+
+:copyright: @ 2018
+:author: gabby ortman
+:license: MIT, see LICENSE
+"""
+from pdb import set_trace as bp
+import json
+from pathlib import Path
+from datetime import timedelta, datetime
+from tomaty.lib import utilities
+
+TOMATY_JSON_PATH = "{}/.tomaty.json".format(Path.home())
+
+class TomatySerializer:
+    def __init__(self):
+        self._total_time = None
+        self.current_date = "{}".format(datetime.now().strftime("%Y_%m_%d"))
+
+    @property
+    def total_time(self):
```

Comment:
docstring pls all methods sans init pls, ty i really appreciate it.

Reply:
are you using me for...DOCUMENTATION?

Reply:
_**sweats**_